### DISCUSSION TITLE
SQL advice

### DISCUSSION THREAD

**Discussion by ondohotola** [dathere/qsv#2386] [FAQ]
Given two CSV files with differing headers headers and differing number of records how do I find the ones in the bigger one that are not in the smaller one, using a column (of names) in each?

**Comment by ondohotola** [comment]
figured it out

**Comment by datatraveller1** [comment]
Hi @ondohotola, if you don't mind can you show your solution? 😃

**Comment by ondohotola** [comment]
something like

```sql
SELECT t1.* FROM largertable t1 
WHERE t1.some NOT IN (SELECT t2.another FROM smallertable t2);
```
